ContactsApp/Exception.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Exception.swift
+//  ContactsApp
+//
+//  Created by Rohit Bansal on 04/10/15.
+//
+//
+
+import Foundation
+
+class Exception : NSObject, ErrorType {
```

Comment:
I you want to use the verb validate instead of isValid, then remove the Bool return type from validate method.

Even with this fix, I will not integrate this usage of exception, exceptions shouldn't be use to control the flow when possible.